libcontainer/factory_linux.go

```diff
@@ -365,6 +365,12 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		defer consoleSocket.Close()
 	}
 
+	logPipeFdStr := os.Getenv("_LIBCONTAINER_LOGPIPE")
+	logPipeFd, err := strconv.Atoi(logPipeFdStr)
+	if err != nil {
+		return fmt.Errorf("unable to convert _LIBCONTAINER_LOGPIPE=%s to int: %s", logPipeFdStr, err)
+	}
+
```

Comment:
Alternatively, we can pass logFd as a parameter to `StartInitalization` as we just converted it from env in `init.go`...

Decided to do it this way since all the other parameters are obtained from environment right here.